### PR TITLE
fix: [SRM-13173]: Add metric thresholds to Sumo Logic

### PR DESCRIPTION
**Pull request by praneshg239**
#### Screenshots
image

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/85-cv/pages/health-source/connectors/CommonHealthSource/CommonHealthSource.tsx [hunk 1]
```diff
@@ -124,8 +126,24 @@ export default function CommonHealthSource({
                   )
                 }}
               </Formik>
+              {/* ⭐️ Metric threshold section  */}
+
+              {getCanShowMetricThresholds({
+                isMetricThresholdConfigEnabled: Boolean(healthSourceConfig?.metricThresholds?.enabled),
```
Comment (deepesh-ui) [inline src/modules/85-cv/pages/health-source/connectors/CommonHealthSource/CommonHealthSource.tsx:132]: shall we keep it in variable an use .  we should have simpler name  for getCanShowMetricThresholds

File: src/modules/85-cv/pages/health-source/connectors/CommonHealthSource/components/CustomMetricForm/CustomMetricFormContainer.tsx [hunk 2]
```diff
@@ -69,6 +69,7 @@ export default function CustomMetricFormContainer(props: CustomMetricFormContain
   const isConnectorRuntimeOrExpression = getMultiTypeFromValue(connectorRef) !== MultiTypeInputType.FIXED
 
   const [groupNames, setGroupName] = useState<SelectOption[]>(initializeGroupNames(mappedMetrics, getString))
+
```
Comment (deepesh-ui) [inline src/modules/85-cv/pages/health-source/connectors/CommonHealthSource/components/CustomMetricForm/CustomMetricFormContainer.tsx:72]: empty space intentional ?

File: src/modules/85-cv/pages/health-source/connectors/CommonHealthSource/components/MetricThresholds/MetricThresholdProvider.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+import IgnoreThresholdContent from './components/IgnoreThresholdsContent'
+import type { CommonMetricThresholdProviderProps } from '../../CommonHealthSource.types'
+import FailFastThresholdContent from './components/FailFastThresholdsContent'
+
+export default function MetricThresholdProvider(props: CommonMetricThresholdProviderProps): JSX.Element {
+  const { getString } = useStrings()
+  return (
+    <Container margin={{ bottom: 'huge' }} padding="large" data-testid="commonHealthSource_metricThresholds">
+      <Accordion activeId="MetricThresholdAdvanced">
+        <Accordion.Panel
```
Comment (deepesh-ui) [inline src/modules/85-cv/pages/health-source/connectors/CommonHealthSource/components/MetricThresholds/MetricThresholdProvider.tsx:16]: <Accordion.Panel />, we don't need empty <Accordion.Panel><Accordion.Panel/>

File: src/modules/85-cv/pages/health-source/connectors/CommonHealthSource/components/CustomMetricForm/CustomMetricFormContainer.tsx (hunk 2 above)
Comment (praneshg239) [reply src/modules/85-cv/pages/health-source/connectors/CommonHealthSource/components/CustomMetricForm/CustomMetricFormContainer.tsx:72]: which space @deepesh-ui ?

File: src/modules/85-cv/pages/health-source/connectors/CommonHealthSource/CommonHealthSource.tsx (hunk 1 above)
Comment (praneshg239) [reply src/modules/85-cv/pages/health-source/connectors/CommonHealthSource/CommonHealthSource.tsx:132]: `getCanShowMetricThresholds` looks simply only, like in which way you are saying?

File: src/modules/85-cv/pages/health-source/connectors/CommonHealthSource/components/MetricThresholds/MetricThresholdProvider.tsx (hunk 3 above)
Comment (praneshg239) [reply src/modules/85-cv/pages/health-source/connectors/CommonHealthSource/components/MetricThresholds/MetricThresholdProvider.tsx:16]: Done

File: src/modules/85-cv/pages/health-source/connectors/CommonHealthSource/components/CustomMetricForm/CustomMetricFormContainer.tsx (hunk 2 above)
Comment (praneshg239) [reply src/modules/85-cv/pages/health-source/connectors/CommonHealthSource/components/CustomMetricForm/CustomMetricFormContainer.tsx:72]: image

File: src/modules/85-cv/pages/health-source/connectors/CommonHealthSource/CommonHealthSource.tsx (hunk 1 above)
Comment (praneshg239) [reply src/modules/85-cv/pages/health-source/connectors/CommonHealthSource/CommonHealthSource.tsx:132]: Done

**Review by deepesh-ui — APPROVED** [review]

**Comment by sunilgupta-harness** [comment]
It is already approved earlier so force merging it

**Pull request merged**